utils/compare/compare.go

```diff
@@ -0,0 +1,39 @@
+package compare
```

Comment:
I'm not sure we really need a new package for a single file.

Reply:
Ok, but I had to move npm utils to the build utils because otherwise, we will have an import cycle.

Reply:
Got it. In that case, let's leave it as is,